.github/scripts/step-status

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
wondering if we can somehow locate this script in "nm-actions" ...

Reply:
Tried it in nm-actions but it doesn't seem to work. The workflow cannot find the script as it's not in the workspace.